### ISSUE TITLE
[ts][pixi-v8] Issue with resolving imports within the package

### ISSUE THREAD

**Issue by alvov-evo** [EsotericSoftware/spine-runtimes#2685]
It seems that since `@esotericsoftware/spine-pixi-v8` package has `"type": "module"` in it's package.json file, all import paths in it must contain file extension, in this case `.js`.

For example, in `SpinePipe.ts`:
```typescript
import { BatchableSpineSlot } from './BatchableSpineSlot';
```
has to be changed to:
```typescript
import { BatchableSpineSlot } from './BatchableSpineSlot.js';
```

From what I see other spine-runtime packages specify `.js` in their imports, so maybe same approach should be applied for spine-pixi-v8?

In my particular care I get and error from Webpack, which says "<...>the origin is strict EcmaScript Module <...> Add the extension to the request."

Thank you!

**Comment by davidetan** [comment]
We spotted this yesterday too at the end of the day.
I wanted to fix the issue this morning, but you have anticipated me 😁 

Thanks for reporting this, I'm going to fix this right now.

**Comment by davidetan** [comment]
This has been fixed in 1b3194a that is present int release 4.2.66.

**Comment by alvov-evo** [comment]
That was blazingly fast, I can confirm that the issue is solved in 4.2.66, thank you! 🙇‍♂️
